pkg/exporter/sumologicexporter/exporter.go

```diff
@@ -16,15 +16,16 @@ package sumologicexporter
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"net/http"
-	"reflect"
+	"strings"
 
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/consumer/consumererror"
 	"go.opentelemetry.io/collector/consumer/pdata"
 	"go.opentelemetry.io/collector/exporter/exporterhelper"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/extension/sumologicextension"
```

Comment:
shouldn't we use local extension?

Reply:
We do, take a look at the go.mod

https://github.com/SumoLogic/sumologic-otel-collector/pull/3/files#diff-810660f89ab0e3e51263333e86eba41eefc5fdbbedc135bcdb2dc3bf115e6455R13